openedx/features/course_experience/__init__.py

```diff
@@ -1,17 +1,28 @@
 """
 Unified course experience settings and helper methods.
 """
-
 import waffle
 
+from openedx.core.djangoapps.waffle_utils import WaffleFlagPlus
 from request_cache.middleware import RequestCache
 
+# Waffle flag to enable the full screen course content view along with a unified
+# course home page.
+# NOTE: This is the only legacy flag that does not use the namespace.
+UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
+
+# Namespace for the waffle flags that follow.
+WAFFLE_NAMESPACE = u'course_experience'
+
 # Waffle flag to enable a single unified "Course" tab.
 UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
 
-# Waffle flag to enable the full screen course content view
-# along with a unified course home page.
-UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
+
+def course_experience_config():
```

Comment:
As mentioned before, I was thinking of something like:

    UNIFIED_COURSE_EXPERIENCE_FEATURE_FLAG = FeatureFlag('unified_course_experience')